src/v3/src/components/Form/Layout.tsx

```diff
@@ -45,7 +51,7 @@ const Layout: FunctionComponent<LayoutProps> = ({ uischema, stepName }) => {
     >
       {
         elements.map((element, index) => {
-          const elementKey = [element.type, stepName, index].join('_');
+          const elementKey = getElementKey(element, index);
```

Comment:
I notice `stepName` was removed from `LayoutProps`. Is it not useful as part of the `key`?

Reply:
Ah, `stepName` is still used, but I had to move where it was being populated. When it was set as a prop to `Layout.tsx` it was causing a weird re-rendering issue when the new transaction gets populated. This issue impacted the WebAuthN flow causing it to display the browser prompt after already signing in. 

I created a new transformer that will append the stepName to the key before it hits layout. This way, if/when we make updates in the future (i.e. if we want to generate some unique GUID per transaction) we can easily append it to each element.